packages/sit-onyx/src/components/OnyxTable/types.ts

```diff
@@ -1,5 +1,12 @@
 import type { DensityProp } from "../../composables/density";
 
 export type OnyxTableProps = DensityProp & {
-  // TODO: add props as needed
+  /**
+   * Whether to use striped row background colors.
+   */
+  striped?: boolean;
+  /**
+   * If `true`, both horizontal and vertical borders will be displayed.
```

Comment:
maybe this could help to make clear that we always have horizontal borders
```suggestion
   * If `true`, vertical borders will be displayed in addition to the default horizontal borders.
```